lib/rules/no-invalid-double-slash-comments/index.mjs

```diff
@@ -47,13 +48,15 @@ const rule = (primary) => {
 					const comment = lines[0];
 					const word = lines.length > 1 ? `${comment}\n` : comment;
 
+					assert(word);
```

Comment:
This is purely because TypeScript always assumes that `arr[0]` can be nullish right?

Reply:
I have the same question. And, instead of `word`, reusing `comment` seems better to me like this:

```js
let comment = lines[0] ?? '';
if (lines.length > 1) comment += '\n';

endIndex: i + comment.length,
```

Reply:
At this point it should be string that's why I used an `assert`.
If it's not we have a problem.

I don't mind replacing the assert with `?? ''`.